ansible/roles/druid-ingestion/defaults/main.yml

```diff
@@ -1,8 +1,13 @@
 remote_user: druid
+remote_group: users
```

Comment:
Do we need a remote_group called `users` defined here?

Reply:
not specifying a group on some hardened systems (NIC servers) leads to the files being owned by `druid:root` which causes permission issue on those systems

Reply:
removed, not needed